doc/conf.py

```diff
@@ -81,7 +81,7 @@
 
 # General information about the project.
 project = u'Apache Traffic Server'
-copyright = u'2016, dev@trafficserver.apache.org'
+copyright = u'2017, dev@trafficserver.apache.org'
```

Comment:
Should we just import one of the date/time modules and grab the year from there?